include/aspect/vofinterface/vof_system.h

```diff
@@ -0,0 +1,148 @@
+/*
+ Copyright (C) 2016 by the authors of the ASPECT code.
```

Comment:
also, if the directory already contains the name `vof`, no need to duplicate this in the name of the file.